netaddr.go

```diff
@@ -632,6 +632,23 @@ func (ip IP) IsLinkLocalMulticast() bool {
 	return false // zero value
 }
 
+// IsGlobalUnicast reports whether ip is a global unicast address.
+// If ip is the zero value, it will return false.
```

Comment:
I think this second sentence can be dropped. That's pretty implicit, no? I don't think we say that for all the other methods where it's also true.

Reply:
We do say it for the others at the moment, see docs starting at https://pkg.go.dev/inet.af/netaddr#IP.IsInterfaceLocalMulticast.